UserGuide.md

```diff
@@ -593,6 +596,26 @@ Once the GitHub action processes the comment and sends the message, it will be p
 ![Example GitHub Comment](images/Example_GitHub_Comment.png)
 
 
+### <a name="starting-a-pr-comparison-run-from-github"/> **Starting a PR Comparison Run from GitHub**
+
+There is another type of run that can be triggered in CARROT from GitHub and that is the PR Comparison Run.  A PR Comparison run triggers two runs of a test, one with a docker image generated from the head commit of the PR and one with a docker image generated from the base commit of the PR.  Triggering it is also exactly the same is triggering a single run:
```

Comment:
This makes sense as a useful thing, but , it might also make sense to offer the ability to trigger 1 on the head of master and the other on the merged commit of the pr on top of master which is what our test suite runs on.  

The ability to trigger comparisons between any 2 arbitrary git hashes seems pretty useful.

Reply:
That is definitely something that is planned for the future